languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/pt-BR/style.xml

```diff
@@ -952,6 +955,154 @@ USA
+                <example correction="seu">Onde está o <marker>teu</marker> pai?</example>
+                <example correction="suas">Cuidado com as <marker>tuas</marker> maneiras.</example>
+            </rule>
+
+            <rule>
+                <pattern>
+                    <token>contigo</token>
+                </pattern>
+                <message>No português brasileiro, o uso da segunda pessoa do singular não é consistente em todos os dialetos. Seu texto soará mais neutro se substituir por <suggestion>com você</suggestion>.</message>
+                <example correction="com você">Posso sentar <marker>contigo</marker>?</example>
+            </rule>
+
+            <rule>
+                <antipattern>
+                    <token postag_regexp="yes" postag="SP.+"/>
+                    <marker><token>ti</token></marker>
+                    <token regexp="yes" inflected="yes">mesmo|próprio</token>
+                </antipattern>
+                <pattern>
+                    <token postag_regexp="yes" postag="SP.+"/>
+                    <marker><token>ti</token></marker>
+                </pattern>
+                <message>No português brasileiro, o uso da segunda pessoa do singular não é consistente em todos os dialetos. Seu texto soará mais neutro se substituir por <suggestion>você</suggestion>.</message>
+                <example correction="você">Pensei logo em <marker>ti</marker>.</example>
+            </rule>
+
+            <rule>
+                <pattern>
+                    <marker>
+                        <token>ti</token>
+                    </marker>
+                    <token regexp="yes" inflected="yes">mesmo|próprio</token>
+                </pattern>
+                <message>No português brasileiro, o uso da segunda pessoa do singular não é consistente em todos os dialetos. Seu texto soará mais neutro se substituir por <suggestion>si</suggestion>.</message>
+                <example correction="si">Pensa em <marker>ti</marker> mesmo antes de tudo!</example>
+            </rule>
+
+            <rule> <!-- reflexive proclitic -->
+                <pattern>
+                    <token>te</token>
+                    <token postag_regexp="yes" postag="V...2S."/>
+                </pattern>
+                <message>No português brasileiro, o uso da segunda pessoa do singular não é consistente em todos os dialetos. Seu texto soará mais neutro se substituir por &quot;te&quot;.</message>
+                <suggestion>se <match no="2" postag_regexp="yes" postag="(V...)2S(.)" postag_replace="$13S$2"/></suggestion>
+                <example correction="se esqueça">Não <marker>te esqueças</marker>!</example>
+                <example correction="se esqueceu">Acho que <marker>te esqueceste</marker>...</example>
+            </rule>
```

Comment:
As per the [diff](https://regression.languagetoolplus.com/via-http/2023-06-30/pt-BR/result_style_INFORMAL_2SG_BR%5B7%5D.html), all conjugation correction rules (`AUX_VERBO`, `CONDITIONAL_NO_INF`, `GENERAL_VERB_AGREEMENT_ERRORS`, `AUXILIARY_VERB_INFINITIVE`) should be prioritized over this one. (I'd argue perhaps all grammar rules should be prioritized over style ones 🤔)

Reply:
That's very true, rule ordering in this kind of case is crucial.